dotCMS/src/integration-test/java/com/dotcms/datagen/TestDataUtils.java

```diff
@@ -2290,5 +2290,71 @@ public static ContentType newContentTypeFieldTypesGalore(final String contentTyp
         return contentType;
     }
 
+    public static ContentType newContentTypeWithMultipleCategoryFields(final String contentTypeName,
+            final Set<String> workflowIds) {
+        final CategoryAPI categoryAPI = APILocator.getCategoryAPI();
+        final List<Category> categories = Try.of(
+                        () -> categoryAPI.findTopLevelCategories(APILocator.systemUser(), false))
+                .getOrElseThrow(DotRuntimeException::new);
+        return newContentTypeWithMultipleCategoryFields(contentTypeName, workflowIds, categories);
+    }
+
+    @WrapInTransaction
+    public static ContentType newContentTypeWithMultipleCategoryFields(final String contentTypeName, Set<String> workflowIds, final List<Category> parents) {
+
+        ContentType contentType = null;
+        try {
+            try {
+                contentType = APILocator.getContentTypeAPI(APILocator.systemUser()).find(contentTypeName);
```

Comment:
Would be better use Try.of().getOrNull

Reply:
good observation. I was just reusing code from other methods in this class

Reply:
done